windows-service-guide.md

```diff
@@ -0,0 +1,78 @@
+# Cloud SQL Auth Proxy Windows Service Guide
+
+This document covers running the *Cloud SQL Auth Proxy* as service
+on the Windows operating system.
+
+It was originally built and tested using Go 1.20.2 on Windows Server 2019.
+
+## Install the Windows Service
+
+Prerequisites: A built binary for Windows of the Cloud SQL Auth Proxy is required. Either build it from source or [download a release](https://github.com/GoogleCloudPlatform/cloud-sql-proxy/releases) of a Windows pre-built version, e.g. `cloud-sql-proxy.x64.exe`. 
+
+First, install the binary by:
+
+1. Create a new empty folder, e.g. `C:\Program Files\cloud-sql-proxy`
+2. Copy the binary and helper batch files
+3. Grant *read & execute* access to the `Network Service` user
```

Comment:
In my testing, I had to add Network Service to the parent folder as well, but that was because I ran out of my user directory. Should we add a note here about that?